Purchasing.Mvc/Controllers/AdminController.cs

```diff
@@ -573,7 +589,7 @@ public ActionResult ValidateChildWorkgroups()
         public virtual bool NeedToCheckWorkgroupPermissions(string key)
```

Comment:
We don't call this from anywhere anymore. It is very database intensive.
Possibly we should remove it.